modules/ichorcna/createpon/main.nf

```diff
@@ -0,0 +1,46 @@
+def VERSION = '0.3.2' // Version information not provided by tool on CLI
+
+process ICHORCNA_CREATEPON {
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::r-ichorcna=0.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/r-ichorcna:0.3.2--r41hdfd78af_0' :
+        'quay.io/biocontainers/r-ichorcna:0.3.2--r41hdfd78af_0' }"
+
+    input:
+    path(wigs)
+    path gc_wig
+    path map_wig
+    path centromere
+
+    output:
+    path "*.rds"        , emit: rds
+    path "*.txt"        , emit: txt
+    path "versions.yml" , emit: versions
+
+    script:
+    def args = task.ext.args ?: ''
+    def centro = centromere ? "--centromere ${centromere}" : ''
+
```

Comment:
Looks like the output can be named. Does anything speak against adding a prefix notation hree?

Reply:
This function generates a file representing the average output of the panel of normals, which can then be used in the main run script. You feed it any number of wig files (each from a bam file). The name of the output file is controlled with the `--outfile` option, which I currently just put to "PoN", but that could be whatever you want to set it to. Not sure how best to configure that though.

Reply:
https://github.com/nf-core/modules/blob/master/modules/gatk4/createsomaticpanelofnormals/main.nf maybe like this, but instead of 

```
    def prefix = task.ext.prefix ?: "${meta.id}"
```
this:
```
    def prefix = task.ext.prefix ?: "PoN"
```

Is the file ending automatically added?

Reply:
and then `--output ${prefix}`

Reply:
Yes, the file ending is automatically added, just needs specifying. Ok, so that lets people set that to what they want, with a default if empty.